src/screens/shared/FarmSearch.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react';
+import Modal from 'react-modal';
+
+import '../../styles/FarmSearch.css';
+import FarmCard from './components/FarmCard';
+import NewFarmCard from './components/NewFarmCard';
+
+import SearchIcon from '../../assets/search-icon.png';
+import farmProfileDefaultCover from '../../assets/farmProfileCover.png';
+import farmProfileDefaultIcon from '../../assets/farmProfilePhoto.png';
+
+const ROOT_ELEMENT = '#root';
+Modal.setAppElement(ROOT_ELEMENT);
+
+class FarmSearch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showFarmModal: false
+    };
+  }
+
+  showModal = () => {
+    this.setState({ showFarmModal: true });
+  };
+
+  hideModal = () => {
+    this.setState({ showFarmModal: false });
+  };
+
+  render() {
+    const { showFarmModal } = this.state;
+    return (
+      <div className="farmsearch-container">
+        <div className="farmsearch-header">
+          Farm Search
+          <div className="farmsearch-box">
+            <input type="text" placeholder="Search" name="search" />
+            <input type="image" src={SearchIcon} alt="searchIcon" />
+          </div>
+          <button type="button" className="farmsearch-filterbox">
+            GAP Certification Stage
+          </button>
+          <button type="button" className="farmsearch-filterbox">
+            Sort By:
+          </button>
+        </div>
+
+        <button
+          type="button"
+          className="buttontest"
+          onClick={() => this.showModal()}
+        >
+          Overlay Test
+        </button>
+        <Modal
+          isOpen={showFarmModal}
+          onRequestClose={() => this.hideModal}
+          className="farm-modal"
+          overlayClassName="farm-modal-overlay"
+        >
+          Farm Profile
+          <button
+            type="button"
+            className="buttontest"
+            onClick={() => this.hideModal()}
+          >
+            Close
+          </button>
+          {/* 
+        insert Phoebe's card code here
+        */}
+        </Modal>
+
+        <div className="farmsearch-gridcontainer">
+          <NewFarmCard />
+          <FarmCard
+            hubApproved="HUB"
+            S
+            gapApproved="GAP"
```

Comment:
Pass `hubApproved` and `gapApproved` in as booleans and use conditional statement in the card to render it (or not render it). This is likely how it will be stored in the backend.

Reply:
done